server/src/query_tests/influxrpc/read_group.rs

```
@@ -133,8 +133,8 @@ async fn test_read_group_data_field_restriction() {
         "+-------+--------+-------+------+",
         "| state | city   | temp  | time |",
         "+-------+--------+-------+------+",
-        "| CA    | LA     | 180   | 550  |",
-        "| MA    | Boston | 142.8 | 350  |",
+        "| CA    | LA     | 180   | 350  |",
+        "| MA    | Boston | 142.8 | 250  |",
```

Got it. `Sum` is now considered as `Max`. I wonder should we throw an error message instead? I see you added a comment above saying  setting `Max` make it consistent with the Go implementation but I feel like this does not make sense

---

@NGA-TRAN the behaviour here is only on the `influxrpc` frontend though, so the Go implementation is the correct thing to do, because that's what we need to replicate. 

You can't apply _any_ aggregate to timestamps in InfluxDB but they still implicitly have to get aggregated so there is one value per returned group. That's always the `max` value of the timestamp in the group (although I still have a feeling it might be `min` in some cases).

---

> we do not support Sum on timestamp

@NGA-TRAN  At this point I don't see a path to supporting SUM for timestamps in SQL either as it isn't supported in DataFusion and may never be (Postgres doesn't support `SUM`ing timestamps either) as the sum of timestamps doesn't really mean anything. There is more detailed discussion on https://github.com/apache/arrow/pull/9970#issuecomment-817279890 where I proposed adding SUM(timestamp) to DataFusion :)


> although I still have a feeling it might be min in some cases

@e-dard  Your feeling of a disturbance in the force does worry me.... But not enough to prevent merging of this PR.